admin_server/src/main/java/com/backend/admin_server/image_verification/controller/ImageRetrievalController.java

```diff
@@ -0,0 +1,29 @@
+package com.backend.admin_server.image_verification.controller;
+
+import com.backend.admin_server.image_verification.service.ImageRetrievalService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+@Controller
+public class ImageRetrievalController {
+
+    private final ImageRetrievalService imageRetrievalService;
+
+    public ImageRetrievalController(ImageRetrievalService imageRetrievalService) {
+        this.imageRetrievalService = imageRetrievalService;
+    }
+
+    @PostMapping("/verify-image")
```

Comment:
So I get the intent here, but this isn't very RESTful. 

In REST, the resource is central to the API. So what would the resource be in this case? The goal is an access request that is either valid or invalid. The fact that we are using images rather than passwords is secondary to that. 

So I think the API should be `access_request` or similar. To create a new resource in REST, the PUT http method is used, but post is fine.

We'll also need more than just the image from the body. I think at least something to identify what client it is (so we know what they are accessing) and who the user is supposed to be.

Reply:
changed to reflect access requests. Just waiting to finalize the requests or access request model and work off of that. for now this should help the front end test out sending an image